src/pin-code/__tests__/pin-code-mask.scenario.js

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
I recommend adding an 'interaction' to [this file](https://github.com/uber/baseweb/blob/master/vrt/config.js#L11) to prepare the component before taking a screenshot. The interaction should type into the pin-code and we should see the mask displayed in the screenshot